packages/api/src/main/logging/Logging.ts

```diff
@@ -4,34 +4,181 @@
  * This source code is licensed under the Apache-2.0 license found in the
  * LICENSE file in the root directory of this source tree.
  */
-import { Logger } from 'winston'
+import chalk from 'chalk'
 
-import { getRunMode } from '../run-mode'
-import { getWinstonLogger } from './winston-utils'
+import { EventMessage, MessagingClient } from '../messaging'
+import { DEBUG } from '../messaging/constants' // TODO: fix this
```

Comment:
what is this referring to? can we do a more explicit comment?

Reply:
haha, this is another reminder to myself of a var that should be pulled from the common infrastructure for envvars that you've started building out.